keyboards/handwired/dactyl_manuform/dmote/62key/keymaps/default/keymap.c

```diff
@@ -0,0 +1,152 @@
+#include "62key.h"
+#include "rgblight.h"
+#include <keymap_colemak.h>
+#include <sendstring_colemak.h>
+
+extern keymap_config_t keymap_config;
+
+#define MODS_SHIFT_MASK (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT))
+#define MODS_CTRL_MASK (MOD_BIT(KC_LCTL) | MOD_BIT(KC_RCTRL))
+#define MODS_ALT_MASK (MOD_BIT(KC_RALT) | MOD_BIT(KC_LALT))
+
+// Layer ID:
+#define _QWERTY 0   // OS-side Colemak. Default.
+#define _COLEMAK 1  // Keyboard-side Colemak. Portability, emergency.
+#define _NUMERIC 3
+
+// Shorthand:
+#define LAYER_N MO(_NUMERIC)
+#define LAYER_C TG(_COLEMAK)
+#define PASTE LSFT(KC_INS)  // Terminal-compatible paste.
+#define SLQ RALT(KC_9)  // Single left-side quotation mark (in Colemak).
+#define SRQ RALT(KC_0)
+#define EMDASH RALT(LSFT(KC_MINUS))
+#define BK_LCTL CTL_T(KC_LBRACKET)
+#define BK_RCTL CTL_T(KC_RBRACKET)  // Ideally this’d be RCTL.
```

Comment:
You can get a Right Ctrl version of this with:

```suggestion
#define BK_RCTL RCTL_T(KC_RBRACKET)
```